kustomize/kustomize.go

```diff
@@ -300,3 +305,20 @@ func GetUnstructuredObjects(data []byte) ([]runtime.Object, error) {
 
 	return items, nil
 }
+
+func BytesToUtf8Lf(file []byte) (string, error) {
+	decoded := utfutil.BytesReader(file, utfutil.UTF8)
+	buf := new(bytes.Buffer)
+	bytesRead, err := buf.ReadFrom(decoded)
+	if err != nil {
+		logger.Errorf("error reading from buffer. bytesRead %v. err: %v", bytesRead, err)
```

Comment:
Another question: why would we need to put `bytesRead` in the log?
Won't the error suffice?

Reply:
I'm not doing anything with it so just thought it might enrich the log in a useful way.

Reply:
It'll make it harder to understand the logs.... the bytes will be present in the logs like:
```
[20 50 ...]
```
If not nil in the first place^
Which is not understandable.
I suggest we should just put err in the logs

Reply:
It's an int value not the actual bites read. Just a total.

Reply:
Okay... so what value would it provide?

Reply:
Well, 0 would be significant as the error occurring at the start of reading whereas some non-zero number would be interesting because it would, I think, mean that the error happened while parsing. 

It's not a huge benefit but then the cost of having it is pretty low.

Reply:
@moshloop what do you think about this?

Reply:
Pretty sure bytesRead will be 0 if there is an issue - either way it isn't useful in the logs

Reply:
From what I can tell in the docs it will report bytes read even if there is an error: https://pkg.go.dev/bytes#Buffer.ReadFrom

I'll take it out of that's the consensus though.

Reply:
Amended.